deployment/src/main/java/io/quarkiverse/githubapp/deployment/devui/GitHubAppDevUIProcessor.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkiverse.githubapp.deployment.devui;
+
+import io.quarkus.deployment.IsDevelopment;
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.devui.spi.page.CardPageBuildItem;
+import io.quarkus.devui.spi.page.ExternalPageBuilder;
+import io.quarkus.devui.spi.page.Page;
+
+/**
+ * Dev UI card for displaying important details such as the GitHub App Replay UI.
+ */
+public class GitHubAppDevUIProcessor {
+
+    @BuildStep(onlyIf = IsDevelopment.class)
+    void createVersion(BuildProducer<CardPageBuildItem> cardPageBuildItemBuildProducer) {
+        final CardPageBuildItem card = new CardPageBuildItem();
+
+        String replayUrl = "http://localhost:8080/replay/";
```

Comment:
I think we need to go with `/replay/` so that it works even if we change the port?

Otherwise, it looks good, thanks!

Reply:
Hmm because the DevUI Card makes it a link i think it has to be a full URL and not a relative one.  I could be wrong lets ask @phillip-kruger ?

Reply:
Relative URL should work. See https://github.com/quarkusio/quarkus/blob/cdc1637bebb289320c36c2c859691746751e65c8/extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/devui/SmallRyeGraphQLDevUIProcessor.java#L21C16-L21C26 as an example.

Reply:
Thanks @phillip-kruger i updated my PR with that!